functional-tests/src/test/java/com/redhat/che/functional/tests/WorkspaceTest.java

```diff
@@ -59,10 +59,12 @@ public void testCorrectWayOfStarting() throws Exception {
   @Test
   public void testWrongWayOfStarting() throws ExecutionException, InterruptedException {
     LOG.info("Try to start " + firstWorksapce.getName() + " without PATCH.");
+    LOG.info("--- ERROR EXPECTED ---");
```

Comment:
What's the purpose of these two log outputs? 
If you want to refactor this method, you are more than welcome, but do it in separate issue... There are definitely things to improve here (order of methods is fixed - if they would run in different order, they will fail. Also I think it would be better to use expectedException, than own handling of exception... And another thing is to use more specific exception than just Exception (this boils further down the code to fix)).
I don't see any value in adding such logs into this method... I think this method deserves some more refactoring (but unfortunately we have more important tasks on our hands)

Reply:
This test case was putting an entire stack trace into stdout, because of method that it uses, so I added the `error expected` test, so that we won't be intrigued by the exception (which it is supposed to throw)

Reply:
I still don't see a point in having such log message here. It solves nothing. This method has bigger problems than this and should (IMHO) be refactored. But that should be done as part of another issue/PR.

Reply:
ok

Reply:
agree with @rhopp those terminal approach log entries should be avoided